Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -845,6 +845,17 @@ def test_bracketed_paste_single_line(self):
 
 @skipUnless(pty, "requires pty")
 class TestMain(TestCase):
+    def setUp(self):
```

Comment:
Please add a comment explaining why your remove these env vars with a reference to the issue.

Reply:
done